### ISSUE TITLE
Back issues show date format syntax (Y-m-d) instead of date

### ISSUE THREAD

**Issue by NateWr** [pkp/pkp-lib#7814]
**Describe the bug**
The date for a back issue is shown as `Y-m-d` instead of the date.

![issues](https://user-images.githubusercontent.com/2306629/161107179-079f7352-af62-48eb-be91-5180ca87fae8.png)

**To Reproduce**
1. Go to Issues and publish an issue.
2. Go to Issues > Back Issues
5. See error

**What application are you using?**
`main` branch (3.4 pre-release)

**Additional information**
@asmecher this is probably related to the recent work on strftime. Did that effect `stable-3_3_0`?

**Comment by asmecher** [comment]
@NateWr, are you switching between branches with the same configuration file?

If your installation was done with pre-3.4.x, your `config.inc.php` will be using [`Y-m-d` style](https://www.php.net/manual/en/datetime.format.php) format options. These are only supported with 3.4.0 and up.

Configuration files created with 3.3.x and older will use [`%Y-%m-%d` style formats](https://www.php.net/manual/en/function.strftime.php) and with 3.4.x these will be backward-compatible.

tl;dr: 3.4.x supports both formats but 3.3.x doesn't.

**Comment by NateWr** [comment]
@asmecher it looks like `IssueGridCellProvider` still uses `strftime`. Is that intended?

https://github.com/pkp/ojs/blob/main/controllers/grid/issues/IssueGridCellProvider.inc.php#L91

**Comment by asmecher** [comment]
No, thanks -- I've fixed it. Otherwise is this issue resolved?
